src/main.jsx

```diff
@@ -28,7 +28,7 @@ root.render(
       <Route
         path="/normalizationDashBoard"
         element={
-          <div className="h-screen">
+          <div>
```

Comment:
if you need a div just to wrap two other elements, consider using Fragments `<>`

Reply:
done